### DISCUSSION TITLE
How to serve a specific IP for a specific FQDN?

### DISCUSSION THREAD

**Discussion by wsw70** [AdguardTeam/AdGuardHome#4610] [Q&A]
I just set up Adguard Home for my domain `example.eu`. 

I have on my network devices that will request `mqtt.example.info` and I would like to always return them `192.168.10.2`

I thought that using *DNS rewrites* would be the solution so I created a rewrite for `mqtt.example.info` (did not work), and then `*.example.info` (did not work either). 

Is this the way to go, or should I directly edit the configuration file, or isn't it possible at all?

**Comment by ainar-g** [comment]
The _DNS rewrites_ feature should work, and if it doesn't, there may be a configuration mistake in your devices of client settings.

But a better way to do that is through the [`dnsrewrite` rules][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite
